app/notifications/process_client_response.py

```diff
@@ -83,7 +85,9 @@ def _process_for_status(notification_status, client_name, reference):
     service_callback_api = get_service_callback_api_for_service(service_id=notification.service_id)
 
     if service_callback_api:
-        send_delivery_status_to_service.apply_async([str(notification.id)], queue=QueueNames.CALLBACKS)
+        encrypted_notification = create_encrypted_callback_data(notification, service_callback_api)
+        send_delivery_status_to_service.apply_async([str(notification.id), encrypted_notification],
+                                                    queue=QueueNames.CALLBACKS)
```

Comment:
This does the same thing as _check_and_queue_callback_task(notification) in notifications_ses_callback. Should we combine them?

Reply:
Perhaps but that will take 2 deploys to remove it. I thought since it doesn't harm anything nor change the functionality I might leave it.